tasks/27_basic-subclasses/basic-subclasses.js

```diff
@@ -0,0 +1,44 @@
+export class God {
+  static create() {
+    let humans = [];
+    humans.push(new Man('Adam'), new Woman('Eve'));
+    return humans;
+  }
```

Comment:
I understand that this is kinda ideological question, but don't we have to create a God's instances?)
You can implement singleton here, but static methods like this is not really good design approach.

Reply:
@ArtemLunt  spec says so
`God.create()`